tauri/src/app.rs

```diff
@@ -199,7 +203,7 @@ pub struct AppBuilder<A: ApplicationExt, C: AsTauriContext> {
   webviews: Vec<Webview<A>>,
 }
 
-impl<A: ApplicationExt + 'static, C: AsTauriContext> AppBuilder<A, C> {
+impl<A: ApplicationExt + 'static, C: AsTauriContext> AppBuilder<C, A> {
```

Comment:
It's not really that important but it seems like it'd be best to switch the type defs too so it isn't backwards.

Reply:
i didn't really focus on it because it's going away during #1226

Reply:
Oh cool!